### ISSUE TITLE
How to set colors when running vim inside Terminal on OS X?

### ISSUE THREAD

**Issue by symbiat** [Yggdroot/indentLine#101]
I tried setting all the colors using your examples in my vimrc but the colors never change:
![screen shot 2015-01-15 at 1 55 13 pm](https://cloud.githubusercontent.com/assets/59488/5764221/59decf16-9cbe-11e4-9a10-e666ad5d7600.png)

**Comment by symbiat** [comment]
My .vimrc contains:

    set list lcs=tab:\|\
    let g:indentLine_color_term = 239
    let g:indentLine_color_gui = '#A4E57E'
    let g:indentLine_color_tty_light = 7
    let g:indentLine_color_dark = 1

But those settings are being ignored (I can comment them out and the colors remain unchanged).

**Comment by symbiat** [comment]
After some research: looks like the version of GNU screen that ships with MacOS X doesn't support 256 colors even though Terminal does. Inside screen there's only 8 colors to play with. (The only other option is to build my own local copy of GNU screen with a configuration flag enabling 256 colors).

**Comment by symbiat** [comment]
I have installed a copy of screen with 256 colors enabled (can verify by using a script to show all colors). I am still unable to set the color for lcs.

**Comment by Yggdroot** [comment]
Do you have `set t_Co=256` in your vimrc? what colorscheme are you using?

**Comment by symbiat** [comment]
I tried `set t_Co=256` in my `.vimrc` and re-enabled your suggested color values and see no changes. I have not explicitly selected any colorscheme - Im using whatever defaults stock vim binary uses for syntax highlighting on Mac OS X. My Terminal preferences set terminal type to xterm-256color. Im running a version of screen from Homebrew which supports 256 colors. I ran a vim script to display all the colors to check that, you see that in the right hand window in this screenshot:
![screen shot 2015-08-12 at 4 22 18 pm](https://cloud.githubusercontent.com/assets/59488/9235808/128442fe-410f-11e5-9494-1c75a1e051c1.png)
